openshift/openshift-ansible-contrib
# Allow the specification of server group policies when provisioning openstack

#### What does this PR do?
Allow the specification of master and infra server group policies when provisioning openstack

#### How should this be manually tested?

Uncomment the following parameters in all.yml:

    # Specify server group policies for master and infra nodes. Nova must be configured to
    # enable these policies.
    openstack_master_server_group_policies: [affinity]
    openstack_infra_server_group_policies: [affinity]

You can specify 'affinity' or 'anti-affinity'. Run OpenStack provisioning; it should complete with no errors.

#### Is there a relevant Issue open for this?
N/A

#### Who would you like to review this?
cc: @tomassedovic @bogdando @Tlacenka PTAL
